### ISSUE TITLE
AutoVolume does not work under in Android 10 Skoda Fabia III with M.I.C Head Unit 

### ISSUE THREAD

**Issue by ichbinsichbins** [rumburake/AutoVolume#7]
Hello, The app AutoVolume does not work. I have a head unit M.I.C. AVM8, with PX6. I have already written to M.I.C. (as the seller of the radio). I was advised that I should report the new problem here.

The following is the behavior of the app:

The app is installed correctly. MCU and system is up to date. But AutoVolume does not react to speed changes. The radio should get the speed signal via the CanBus. When I start the app and start the engine, the bar in the upper left corner should react and show the current engine speed. But here nothing happens. When I start driving, the second bar on the left should show the current speed. But also here nothing happens. Thus, the volume can not be controlled. It looks like the app doesn't get any signals from the CanBus or doesn't process them.
In the factory settings, the correct CanBus is set. Other signals which are read from the CanBus by the radio or CanBus adapter work correctly and are displayed correctly. For example open doors, handbrake, mileage etc.)

But if I manually push the slider for speed higher in the app, then the volume is also regulated higher. 

At the effect settings I have of course tested various settings to the maximum. As expected but without success. 

I hope in the video the problem is clear to see.

https://user-images.githubusercontent.com/107299680/173180931-c11a22b1-014d-4729-9e48-d04f82c94c32.mp4

**Comment by rumburake** [comment]
Yes, you are right, my car was Mk2. The CANBus info comes on other ports for the Mk3 and later.
Someone if interested should refactor a bit to isolate the model specific code so the callbacks can be extended to more models.
I have now Android Auto so sadly I am unable to contribute but please feel free to fork.
